pkg/model/application_live_state.proto

```diff
@@ -149,3 +154,37 @@ message CloudRunResourceState {
     // The timestamp of the last time when this resource was updated.
     int64 updated_at = 15 [(validate.rules).int64.gt = 0];
 }
+
+// ECSResourceState represents the state of an ECS service resource.
+message ECSResourceState {
+    enum HealthStatus {
+        UNKNOWN = 0;
+        HEALTHY = 1;
+        OTHER = 2;
+    }
+
+    // The arn of the ECS resource.
+    string id = 1 [(validate.rules).string.min_len = 1];
+    // The sorted list of unique IDs of the owners that depended by this resource.
+    // The owner is another resource that created and managing this resource.
+    repeated string owner_ids = 2;
+    // The sorted list of unique IDs of the parents.
+    repeated string parent_ids = 3;
+    // The unique name for the type of resource.
+    string name = 4 [(validate.rules).string.min_len = 1];
+    // The api version of this resource. Use application's ApiVersion for ECS resources.
+    string api_version = 5 [(validate.rules).string.min_len = 1];
+    // The kind of this resource.
+    string kind = 6 [(validate.rules).string.min_len = 1];
+    // The namespace this resource belongs to.
+    // This is empty for ECS resources.
+    string namespace = 7;
```

Comment:
[Ask] Are these needed? (for UI?) If not needed, let's delete them

Reply:
@ffjlabo 
Thanks, 

- `kind`: Required.  for UI. See image on https://github.com/pipe-cd/pipecd/pull/5021
- `namespace`: Not required 
- `api_version`: Not required

I removed `namespace` and `api_version` in [93edc0d](https://github.com/pipe-cd/pipecd/pull/4979/commits/93edc0da52edf35f11a35a1e9c52140a805a8079).
I added the two for future compatibility, but now I think it's better to add them when actually required.

Reply:
> I added the two for future compatibility, but now I think it's better to add them when actually required.
Thank you! In my opinion, the values which we want to show differ between each platform. So that is nice 👍